Automattic/woocommerce-payments
# Update elligibility flag worflows

#### Changes proposed in this Pull Request
- On account data, force `card_present_eligible` flag to false.









#### Testing instructions





* set card_present_enabled meta to 1 for the current store
* make an api call request to `v3/payments/accounts`
* `card_present_eligible` should be false


<img width="1513" alt="Screenshot 2021-10-25 at 15 56 43" src="https://user-images.githubusercontent.com/82826872/138699263-02d004cd-0523-4dec-af8b-f8a9eec8bd33.png">

-------------------


**Post merge**
